modules/distribution/src/repository/resources/dbscripts/apimgt/mssql.sql

```diff
@@ -201,12 +203,12 @@ CREATE TABLE IDN_IDENTITY_META_DATA (
 
 IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[IDN_THRIFT_SESSION]') AND TYPE IN (N'U'))
 CREATE TABLE IDN_THRIFT_SESSION (
-            SESSION_ID VARCHAR(255) NOT NULL,
-            USER_NAME VARCHAR(255) NOT NULL,
-            CREATED_TIME VARCHAR(255) NOT NULL,
-            LAST_MODIFIED_TIME VARCHAR(255) NOT NULL,
-            TENANT_ID INTEGER DEFAULT -1,
-            PRIMARY KEY (SESSION_ID)
+       		SESSION_ID VARCHAR(255) NOT NULL,
+       		USER_NAME VARCHAR(255) NOT NULL,
+       		CREATED_TIME VARCHAR(255) NOT NULL,
+       		LAST_MODIFIED_TIME VARCHAR(255) NOT NULL,
+       		TENANT_ID INTEGER DEFAULT -1,
+       		PRIMARY KEY (SESSION_ID)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04